ocdaction/ocdaction/templates/dashboard/act/task_edit.html

```diff
@@ -29,8 +37,12 @@
                     {% render_field field placeholder=field.label class+="form-control" %}
                 </div>
             {% endfor %}
-
-            <button class="btn btn-primary pull-left col-xs-2 col-xs-offset-5" type="submit">Save</button>
+            <div class="col-xs-6">
+                <button class="btn btn-primary pull-right" type="submit">Save</button>
+            </div>
+            <div class="col-xs-6">
+                <a class="btn btn-primary pull-left" onclick="return checkCancel()" href="{% url 'task-list' %}">Cancel</a>
+            </div>
```

Comment:
Just a few suggestions:
- we can use `<button>` tag her instead of `<a>` here
- there's a class `btn-danger` that we can use to render a nice red button :) 
- I'd also swap the buttons so the Save is on the right and Cancel on the left side as it follows more good UX practices